detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>[minor] typo: "For an information on the underlying technology see the&nbsp;<a href="https://github.com/microsoft/ComputerVision/raw/e41d1e79495563ac644c65d04ce0e0ab4e50d4f2/detection/notebooks/FAQ.md" target="_blank">FAQ</a>&nbsp;and the&nbsp;<a href="https://github.com/microsoft/ComputerVision/raw/e41d1e79495563ac644c65d04ce0e0ab4e50d4f2/detection/notebooks/01_training_introduction.ipynb" target="_blank">training introduction notebook</a>&nbsp;in this directory."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>